tests/test_benchmarks_web_urldispatcher.py

```diff
@@ -92,11 +98,12 @@ async def handler(request: web.Request) -> NoReturn:
     for count in range(250):
         app.router.add_route("GET", f"/api/server/dispatch/{count}/update", handler)
     app.freeze()
+    router = app.router
```

Comment:
Why all these changes? Seems like it's only accessed once, so assigning to a variable is surely just adding slightly more overhead and reducing conciseness.

Reply:
No, it is accessed in a loop a few lines below,.
```
    async def run_url_dispatcher_benchmark() -> None:
        for _ in range(resolve_count):
            await router.resolve(request)
```
Benchmarks actually measure `run_url_dispatcher_benchmark()` execution, not the whole test.
Preparation time is not counted.